db/migrate/20150111203000_add_akismet_score_to_works.rb

```diff
@@ -0,0 +1,11 @@
+class AddAkismetScoreToWorks < ActiveRecord::Migration
+  def self.up
+    add_column :works, :akismet_score, :boolean
```

Comment:
I'm not sure we should call this "akismet_score" if it's a boolean, since score suggests a number.